test/test_id.cxx

```diff
@@ -14,9 +14,6 @@ class KaxTracks {
 
 int main(void)
 {
-    constexpr binary buf[4] = { 0x12, 0x34, 0x56, 0x78 };
-    EbmlId frombuf(EbmlId::FromBuffer(buf, 4));
```

Comment:
This is a test code. It does not matter if the variable is not used later. We want to make sure it compiles.

That's why using `-Werror` doesn't sound like a good idea because it forces us to clean things we do on purpose. In this case using `[[maybe_unused]]` should fix the issue properly. But it might fordbid merging some code later that is clean.

Reply:
Also in MSVC builds there's a **ton** of warnings. If they are ever turned into errors we're in trouble.

These (mostly) legit warnings are the real main issues with the code at the moment.